src/methods/scprint/config.vsh.yaml

```
@@ -80,10 +80,10 @@ engines:
         run: lamin init --storage ./main --name main --schema bionty
       - type: docker
         run: lamin load anonymous/main
-      - type: python
-        script: from scdataloader.utils import populate_my_ontology; populate_my_ontology()
       - type: python
         script: import bionty as bt; bt.core.sync_all_sources_to_latest()
+      - type: python
+        script: from scdataloader.utils import populate_my_ontology; populate_my_ontology()
 runners:
   - type: executable
```

Add this line to the config file (but don't commit it). That should let Docker see your GPU.

```suggestion
  - type: executable
    docker_run_args: --gpus all
```